src/test/suite/fixtures/recorded/selectionTypes/changeSmallPaint.yml

```diff
@@ -0,0 +1,22 @@
+languageId: plaintext
+command:
+  version: 1
+  spokenForm: change small paint
+  action: clearAndSetSelection
+  targets:
+    - {type: primitive, selectionType: nonwhiteSpaceOrDelimiterSequence}
+initialState:
+  documentContents: "' another '"
```

Comment:
You have a bunch of tests like this, where it's targeting one token.  It's worth having one test like this, but most tests should probably use something with a dash in it or something, eg `foo-bar`, to test the "paint" part of it, rather than focusing only on the "small" part 😊